src/common/Loader/styled.js

```diff
@@ -0,0 +1,17 @@
+import { styled, keyframes } from "styled-components";
+
+export const rotation = keyframes`
+0%{ transform: rotate(0deg);
+}
+100%{transform:rotate(360deg);
+}`;
+
+export const StyledSvg = styled.svg`
+  display: block;
+  max-width: 91px;
+  max-height: 91px;
+  margin: auto;
+  margin-top: 219px;
+  fill: #f5f5fa;
```

Comment:
we use theme features

Reply:
What do you mean you use them features. these are styles for the loader spinner